recipes/pymagicc/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pymagicc" %}
+{% set version = "2.0.0b11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed74ef60e7fda07710bd19bbb3cbf2d9a13998780cc860f80ac3cfbc6a9d00e3
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - pandas-datapackage-reader
+    - f90nml
+    - PyYAML
+    - scmdata >=0.2.1
+
+test:
+  imports:
+    - pymagicc
+
+  commands:
+    python -c "import pymagicc; assert pymagicc.__version__ == '2.0.0-beta.11'"
+
+about:
+  home: https://github.com/openclimatedata/pymagicc
+  license: AGPLv3
```

```suggestion
  license: AGPL-3.0
```
https://opensource.org/licenses/AGPL-3.0

---

Pymagicc wraps the CC-BY-NC-SA licensed MAGICC6 binary. Pymagicc itself is AGPL licensed.
Should this be noted somewhere/somehow in the metadata?